NoComponentDrag.py

```diff
@@ -28,27 +28,28 @@
 import math, os, operator, time
 from collections import deque
 
-NAME = 'NoComponentDrag'
-FILE_DIR = os.path.dirname(os.path.realpath(__file__))
-
 # Import relative path to avoid namespace pollution
 from .thomasa88lib import utils, events, manifest, error
 
 # Force modules to be fresh during development
+# Does not need a "thomana88lib." before the module since import lib has-
+# To RE-import pre-referanced modules above the direct names can be used
+# VSCode intillesense does not like it if you do it with the directory (Gives not referenced error but still functions properly)
 import importlib
```

Comment:
The imports here can be replaced if you create a function in your LibUtils thingy which just needs to be-
`# Allows for re-import of multiple modules
def ReImport_List(*args):
	for module in args:
		importlib.reload(module)`
Thus your code to reimport can be simplified in all of your apps as:
`ReImport_List(utils, events, manifest, error)`